client/code-mirror.js

```diff
@@ -31,10 +31,15 @@ var CodeMirror = React.createClass({
       theme: 'default',
       mode: 'markdown',
       lineWrapping: true,
+      dragDrop: true,
+      autofocus: true,
+      inputStyle: 'textarea',
+      rtlMoveVisually: true
     }
     for (var key in this.props.adminSettings.editor) {
       editorSettings[key] = this.props.adminSettings.editor[key]
     }
+    editorSettings.inputStyle = 'textarea';
```

Comment:
why textarea?
![image](https://cloud.githubusercontent.com/assets/112170/26718409/3717e448-4781-11e7-9554-8d342ce53c17.png)
seems like contenteditable is the future